serde/serde.go

```diff
@@ -0,0 +1,30 @@
+// Copyright © 2023 Meroxa, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package serde
+
+import (
+	sdk "github.com/conduitio/conduit-processor-sdk"
+	"github.com/conduitio/conduit-processor-sdk/internal/proto"
+)
+
+func MarshalCommand(resp sdk.Command) ([]byte, error) {
+	//nolint:wrapcheck // a wrapper function to be used in Conduit
```

Comment:
Oh I see, this is supposed to be used in Conduit. Well this makes me think if we could simplify this by just exposing the `wasm` package instead of having it in the `internal` package. Then we have access to everything we need in Conduit as well, and we can just document it as a package that shouldn't be used in processors.